addons/mail/static/src/messaging/component/message_list/message_list.js

```diff
@@ -276,6 +276,15 @@ class MessageList extends Component {
         ) {
             return false;
         }
+        if (
+            prevMessage.notifications.length > 0 ||
+            message.notifications.length > 0
+        ) {
+            // When messages are squashed, `o_Message_header` is only visible
+            // for the first message. This would prevent from seeing
+            // notifications from the 2nd message onwards.
```

Comment:
Basically could be summarized with "Visual about message notifications is restricted to non-squashed messages."

Reply:
Ok I'll simplify it, I added more details since you said it wasn't clear originally.